src/main/java/net/masterthought/cucumber/json/Embedding.java

```diff
@@ -63,28 +64,35 @@ public String getFileId() {
     }
 
     public String getExtension() {
-        switch (mimeType) {
+        // prepare/ensure switch-case matching
```

Comment:
some reference to official documentation is more than welcome

Reply:
The method now has a JavaDoc documentation.
- https://github.com/damianszczepanik/cucumber-reporting/commit/45261062e90c5fc27294f2e41c2d86fc642bdeeb